tree/ntuple/doc/BinaryFormatSpecification.md

```diff
@@ -681,6 +682,17 @@ The entry range allows for finding the right page list for random access request
 The number of clusters information allows for using consistent cluster IDs
 even if cluster groups are accessed non-sequentially.
 
+#### Linked Attribute Sets
+
+An RNTuple may have zero or more linked Attribute Sets, containing metadata.
+Each Attribute Set is stored on disk as an RNTuple and the Anchor of each RNTuple is linked to by the main
+RNTuple's footer.
+An Attribute Set RNTuple has a number of restrictions compared to a regular RNTuple, most notably it cannot have
+linked Attribute RNTuples itself.
+
+The payload of each list element is a Locator pointing to the Attribute RNTuple's Anchor followed by a string
+containing the Attribute Set's name.
```

Comment:
The locator and the name should be in a record frame, so that we can extend it in the future.